Source/WebCore/platform/Site.h

```diff
@@ -63,6 +64,7 @@ class Site {
 private:
     String m_protocol;
     RegistrableDomain m_domain;
+    String m_string;
```

Comment:
You are not initializing or setting `m_string` anywhere so `m_string` is always null and I am not sure why we need this data member. 
`string()` should just return based on the value of `m_protocol` and `m_domain`

Reply:
I'm setting it in `string()`, it's just lazy initialized. You think we should just make a new string every time?

Reply:
> string() should just return based on the value of m_protocol and m_domain
I made this change.